front/lib/api/assistant/agent.ts

```diff
@@ -48,6 +48,7 @@ import { redisClient } from "@app/lib/redis";
 import logger from "@app/logger/logger";
 
 const CANCELLATION_CHECK_INTERVAL = 500;
+const MAX_ACTIONS_PER_STEP = 16;
```

Comment:
even 16 is quite a lot, maybe 8 would be a better default ?

Reply:
def not strongly held though

Reply:
Since topK are shared I think 16 can be useful to get some breadth int he results